### ISSUE TITLE
Music Stops Playing in Browser

### ISSUE THREAD

**Issue by grravity** [airsonic/airsonic#1597]
## Problem description

*When playing Airsonic through the web interface, the music plays fine for a bit then randomly stops playing. The music timer is stopped and doesn't offer to play just offers to pause as if it's frozen or stuck playing. It seems like it might be related to background processes or something as when I switch back to that tab it seems like it plays after about 15 seconds. Not entirely sure if this is the issue though. I get no errors in the airsonic log though.*

## System information

 * **Airsonic version**: *	10.5.0-RELEASE – November 7, 2019*
 * **Operating system**: *Apache Tomcat/8.5.43, java 11.0.6, Windows 10 (146.9 MB / 238.0 MB)*
 * **Proxy server**: *Caddy*
 * **Client**: *Firefox 68.6.0esr (64-bit)*
 * **Language**: *English/America*

**Comment by Koenvh1** [comment]
Is it a FLAC audio file, and does it only happen in Firefox? If so, it may be related to: https://bugzilla.mozilla.org/show_bug.cgi?id=1528265 (which is fixed in Firefox 76)

**Comment by grravity** [comment]
Yes 99% FLAC, didn't test in other browsers sorry should've. And ok uh is there a way to tell what ESR version it will be pushed in?

**Comment by Koenvh1** [comment]
Likely ESR 78, as far as I can see on 2020-06-30 (https://wiki.mozilla.org/Release_Management/Calendar). You can also fix it by re-encoding your FLAC files, that fixes the issue as well, but I can imagine re-encoding your entire library may be tedious.

**Comment by matthewrt** [comment]
This is possibly the same issue, I'm experiencing
https://github.com/airsonic/airsonic/issues/1921
